app/Models/Store/Order.php

```diff
@@ -280,15 +280,25 @@ private function removeOrderItem(array $params)
 
     private function newOrderItem(array $params)
     {
+        if ($params['cost'] < 0) {
```

Comment:
shouldn't this after the override at username-change?

Reply:
nope